static-assets/ng-views/create-site.html

```diff
@@ -41,23 +41,23 @@ <h2 class="">{{ 'dashboard.sites.create.CHOOSE_A_BLUEPRINT' | translate }}</h2>
             <div class="col-xs-12 col-sm-7 col-md-6">
                 <div ng-attr-id="{{ 'flexslider' + $index }}" class="flexslider mt15 mainslider" ng-init="flexSilderInit('flexslider' + $index)" >
                     <ul class="slides">
-                        <li class="list-gallery" ng-repeat="media in siteBP.blueprint.media.screenshots" onerror="this.src='/studio/static-assets/images/no_image_available.jpg'" >
-                            <img class="gallery-image" ng-src="{{media.screenshot.url}}" alt="{{media.screenshot.title}}" onerror="this.src='/studio/static-assets/images/no_image_available.jpg'">
+                        <li class="list-gallery" ng-repeat="media in siteBP.plugin.media.screenshots" onerror="this.src='/studio/static-assets/images/no_image_available.jpg'" >
+                            <img class="gallery-image" ng-src="{{media.url}}" alt="{{media.title}}" onerror="this.src='/studio/static-assets/images/no_image_available.jpg'">
```

Comment:
`<li>'s` don't have a onerror. Just `<img>` should suffice.

Would be good to move logic to the controller - simply invoke the function from here - the view.